WalletWasabi.Fluent/Converters/StatusBarStateVisibilityConverter.cs

```diff
@@ -12,13 +12,18 @@ private StatusBarStateVisibilityConverter()
 	{
 	}
 
-	object? IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo culture)
+	object? IValueConverter.Convert(object? value, Type targetType, object? parameter, CultureInfo culture)
 	{
-		return (StatusBarState)value == (StatusBarState)parameter;
+		if (value is StatusBarState state && parameter is StatusBarState paramState)
+		{
+			return state == paramState;
+		}
+
+		return null;
```

Comment:
Returning null is not a good practice. The Ui could act like everything is ok but in fact, there is an error. I would rather throw an exception.

Reply:
made consistent with msft / avalonia docs.

https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Data.IValueConverter?view=winrt-22000